lib/team_api/joiner.rb

```diff
@@ -20,6 +20,7 @@ class Joiner
     def self.join_data(site)
       impl = JoinerImpl.new site
       site.data.merge! impl.collection_data
+      impl.create_indexes
```

Comment:
Yeah...especially since reordering things vis a vis the current `_plugins/joiner.rb` in the Hub is what bit me here. Going to try giving that a shot tonight.